ko/security.rst

```diff
@@ -9,7 +9,63 @@ CUBRID 보안
 
 패킷 암호화
 ===========
+패킷 암호화 필요성
+-------------------
+클라이언트와 서버 간에 암호화 연결을 사용하지 않을 경우, 네트워크에 접근 가능한 해커가 모든 트래픽을 감시하고 클라이언트와 서버 간에 주고받는 데이터를 탈취하여 악용할 수 있다.
+이렇게 제3자(중간자)가 정보를 탈취하여 악용하는 것을 중간자 (MITM, Man in the Middle) 공격이라 한다. 이 중간자 공격은 클라이언트와 서버간 연결에 보안 인증 절차를 추가하여 방지할 수 있다.
 
+패킷 암호화 프로토콜
+-------------------------------
+
```

Comment:
how about deleting line ?

Reply:
Title to describe SSL/TLS. It seems necessary.